pkg/actions/nodegroup/create.go

```diff
@@ -46,7 +46,7 @@ func (m *Manager) Create(options CreateOpts, nodegroupFilter filter.NodeGroupFil
 		return err
 	}
 
-	if err := ctl.LoadClusterIntoSpec(cfg); err != nil {
+	if err := ctl.LoadClusterIntoSpec(cfg, ctl.NewStackManager(cfg)); err != nil {
```

Comment:
Do we need to do `ctl.NewStackManager`? one should exist on `m`

Reply:
actually nvm i changed this line in another pr, it will be fixed in the merge conflict 😈